cmd/kubeadm/app/util/pkiutil/testing/testdata/24.rsa.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
thanks for adding these.
they were generated on every test run but nobody included them in their prs.